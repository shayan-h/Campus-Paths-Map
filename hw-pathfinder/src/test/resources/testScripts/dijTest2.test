# Find a path with Dijkstra's algorithm when null

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C

AddEdge simpleGraph A C 10.0

FindPath simpleGraph A B