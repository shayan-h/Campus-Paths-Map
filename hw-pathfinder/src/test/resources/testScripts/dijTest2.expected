# Find a path with Dijkstra's algorithm when null

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph

added edge 10.000 from A to C in simpleGraph

path from A to B:
no path found